### DISCUSSION TITLE
fabric + PDF

### DISCUSSION THREAD

**Discussion by khaledmndev** [fabricjs/fabric.js#7884] [Q&A] [answered]
Hello Devs, hope you are all having a wonderful day.
i have an issue while i add multiple canvases for every every pdf in pdfjs..
is there anyway i can add a canvas per page and when i click on a specific page i can add control or draw over it ?

i hope my question is understandable 

thanks :)

**Comment by ShaMan123** [comment, marked as answer]
> i hope my question is understandable

It is not.
Working repro is a **MUST**
http://fabricjs.com/import-pdf

**Reply by khaledmndev** [reply]
thanks almost near of what i want, do you have any idea on how i can make it like a flipbook?
like 2 pages showing inline and when i click next it shows me the next two pages..

**Reply by ShaMan123** [reply]
I have no idea what a flipbook is.
You probably want to animate it using skewing.
